k8s/postgres/statefulset.yml

```
@@ -0,0 +1,49 @@
+  serviceName: postgres
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        app: postgres
+    spec:
+      containers:
+      - name: postgres
+        image: "postgres:12.3-alpine"
+        env:
+          - name: POSTGRES_DB
```

yamllint (1.35.1)

[warning] wrong indentation: expected 8 but found 10

(indentation)